src/js/components/ToggleGroup/stories/simple.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box, ToggleGroup } from 'grommet';
+import { List, Table, MapLocation } from 'grommet-icons';
+
+const options = [
+  {
+    icon: <List a11yTitle="List view" />,
+    value: 'list',
+  },
+  {
+    icon: <Table a11yTitle="Map view" />,
+    value: 'table',
+  },
+  {
+    icon: <MapLocation a11yTitle="Map view" />,
+    value: 'map',
+  },
+];
+
+export const Simple = () => (
+  <Box gap="large" pad="large">
+    <ToggleGroup
```

Comment:
This is because we are just inheriting the button theme as is and that's how Grommet theme styles buttons. Are we okay with that? We'd discussed not customizing the button styling beyond padding/rounding.

Reply:
The icon only versions are not receiving the purple hover state. Even though we are inconsistent in Button, similar to height, should we be consistent within ToggleGroup?

Reply:
@halocline Do you want us to enforce either the border or no border hover styling across the board for this component?

Reply:
I was thinking the purple border seemed to be more in alignment with the other button hover states. What's your thinking?

Reply:
After discussion with Matt offline, we determined we should not have the purple border (see rationale in PR description under style overrides).